packages/facility-server/__tests__/apiv1/User.test.js

```diff
@@ -340,108 +355,144 @@ describe('User', () => {
   });
 
   describe('User facility methods', () => {
-    let user = null;
     let superUser = null;
-    let noFacilityUser = null;
+    let linkedUser = null;
+    let unlinkedUser = null;
```

Comment:
Personally don't love the term "linked" to describe this concept, it's about access/permission/allowedness and there are words for that, whereas linking could mean a lot of different things.
```suggestion
    let userWithAccess = null;
    let userWithNoAccess = null;
```

Reply:
Honestly for later use this is easier
```suggestion
    let user1 = null;
    let userWithNoAccess = null;
```

Reply:
Yep I can update. Personally not a fan of the userWithNoAccess though because to me that applies that user should never be able to access anything when there are many cases it can

Reply:
Yeah no worries, that's a great point. Go with whatever name you like, I just prefer to avoid the `link` idea

Reply:
Agree and agree! (I didn't see that comment before I posted mine)